mmv1/third_party/terraform/tests/resource_apigee_sharedflow_deployment_test.go

```diff
@@ -109,7 +120,7 @@ resource "google_apigee_environment" "apigee_environment" {
 resource "google_apigee_sharedflow" "test_apigee_sharedflow" {
   name            = "tf-test-apigee-sharedflow"
   org_id          = google_project.project.project_id
-  config_bundle   = "./test-fixtures/apigee/apigee_sharedflow_bundle.zip"
+  config_bundle   = "%{config_bundle}"
```

Comment:
config_bundle isn't in the schema above. Should this not be updating `revision`?

Reply:
good question! sorry, i could provide more context.

this is the `sharedflow` resource, it has a `config_bundle` and when that field is updated, the [output `revision`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/apigee_sharedflow#revision) is updated.

the `sharedflow_deployment` resource (below) deploys the `sharedflow` resource. i'm changing the deployment resource. when the `sharedflow` resource is updated, and `sharedflow.revision` is updated, this deployment should update. however, previously, it was `ForceNew` and we were trying to delete the old deployment and create a new one. But since the old deployment was still tied to the previous revision, it couldn't be deleted. Does that make sense? [this comment ](https://github.com/hashicorp/terraform-provider-google/issues/14530#issuecomment-1542562465)helped me in understanding the flow of things.

Reply:
mm I see. Config generates output revision on this resource. This is taken as input on the shared resource.